app/clients/sms/aws_pinpoint.py

```diff
@@ -0,0 +1,61 @@
+from time import monotonic
+
+import boto3
+import phonenumbers
+from botocore.exceptions import ClientError
+
+from app.clients.sms import SmsClient
+
+
+class AwsPinpointClient(SmsClient):
+    """
+    AWS Pinpoint SMS client
+    """
+
+    def init_app(self, current_app, statsd_client, *args, **kwargs):
+        self._client = boto3.client("pinpoint-sms-voice-v2", region_name="ca-central-1")
+        super(AwsPinpointClient, self).__init__(*args, **kwargs)
+        # super(SmsClient, self).__init__(*args, **kwargs)
+        self.current_app = current_app
+        self.name = "pinpoint"
+        self.statsd_client = statsd_client
+
+    def get_name(self):
+        return self.name
+
+    def send_sms(self, to, content, reference, multi=True, sender=None):
+        pool_id = self.current_app.config["AWS_PINPOINT_POOL_ID"]
+        messageType = "TRANSACTIONAL"
+        matched = False
+
+        for match in phonenumbers.PhoneNumberMatcher(to, "US"):
```

Comment:
The `US` parameter is to say the phone number has an American phone number format?

Reply:
It's used as the default region, ie if you don't start your number with a region code then it assumes it's US / CA.